Unity-Technologies/ScriptableRenderPipeline
# Fix QualitySettings changing without user interaction

### Purpose of this PR
 Fixed [case 1195272](https://issuetracker.unity3d.com/issues/lwrp-the-anti-alias-quality-settings-value-is-changing-without-user-interaction)

### Testing status

**Manual Tests**: Tested with [PotatoBenchmark](https://github.com/Unity-Technologies/PotatoBenchmark)
- [x ] Opened test project + Run graphic tests locally
- [x ] Built a player
- Other: 

**Automated Tests**: TODO (Validate how we can add automation test here)

### How to test
- Open [PotatoBenchmark](https://github.com/Unity-Technologies/PotatoBenchmark).
- Open Scene 001 - Fillrate
- Save Project
- Check files that change in source control.

Expected with this PR:
 - QualitySettings.asset should not have different AA values.

What's happening on master:
 - QualitySettings.asset changes AA value.

---
### Comments to reviewers
URP is changing QualitySettings.antiAliasing because there's no API exposed to let the render pipeline controls swap chain AA. There is work planned to expose api to SRP to control AA, however that work cannot be backported to 2019.4 LTS. This is a fix that will solve majority of cases a change happen and we can backport to 2019.4
 
I've tested the fix before / after on iOS. There's no extra bandwidth cost because the platform has auto-resolve MSAA. For platforms that don't have auto resolve MSAA this might regress performance a little but, however there's no other way to fix it until the AA api on SRP lands.

Below there are GPU captures to see bandwidth cost. Cost is the same on iPhone 6S.
- Master on iPhone 6S.
<img width="719" alt="Screenshot 2020-03-05 at 15 47 33" src="https://user-images.githubusercontent.com/7453395/75995542-e1209580-5efc-11ea-91c2-0cba3b9b282e.png">

- This PR on iPhone 6S
<img width="950" alt="Screenshot 2020-03-05 at 15 39 49" src="https://user-images.githubusercontent.com/7453395/75995558-e67de000-5efc-11ea-9b77-99279f1db224.png">
